cli/prod/harmony/bc.sh

```diff
@@ -19,7 +19,7 @@ PRICE=
 # binance chain configuration and harmony account in binance chain
 CHAIN=Binance-Chain-Tigris
 NODE=https://dataseed5.defibit.io:443
```

Comment:
I think binance chain had updated the NODE as well, check their document online.